browser-extension/addModal.js

```diff
@@ -85,9 +94,49 @@ function addTournesolModal() {
       const currentDisplay = expectedModal.style.display;
 
       if ((event.key === 'Escape' || event.code === 'Escape')
-          && currentDisplay !== 'none') {
-        expectedModal.style.display = 'none';
+          && currentDisplay !== EXT_MODAL_INVISIBLE_STATE) {
+        expectedModal.style.display = EXT_MODAL_INVISIBLE_STATE;
       }
     }
   }
 };
+
+
+/**
+ * Single entry point to display the extension modal.
+ */
+function displayModal() {
+  const modal = document.getElementById(EXT_MODAL_ID);
+  const iframe = document.getElementById(IFRAME_TOURNESOL_LOGIN_ID);
+
+  if (!modal) {
+    return false;
+  }
+
+  const displayModal = function displayModal() {
+    modal.style.display = EXT_MODAL_VISIBLE_STATE;
+    iframe.removeEventListener('load', displayModal);
+  }
+
+  // prevent visual blink while refreshing the iframe
+  iframe.addEventListener('load', displayModal);
```

Comment:
Multiple functions are named `displayModal`. It's probably fine as the inner one uses `const`, but for readability a different name would help.

Reply:
Thanks!

Fixed by https://github.com/tournesol-app/tournesol/pull/508/commits/0beb7769e92478dc6c157e9111a660cebf7bedbc